service/src/main/java/com/ge/predix/acs/zone/management/dao/AttributeConnectorEntity.java

```diff
@@ -0,0 +1,50 @@
+package com.ge.predix.acs.zone.management.dao;
+
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "attribute_connector")
+public class AttributeConnectorEntity {
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "refresh_interval_hours", nullable = true)
+    private int refreshIntervalHours;
```

Comment:
lets discuss 
* I suggested this to force a minimum cache time of 1 hour as a design pre-req. 
* To provide flexibility, we can use minutes and enforce a minimum of 60 mins. In case someone doesn't care about latency. Default should be probably 8 hours.
* on a side not `maxCacheIntervalMintues` maybe  more clear as a name